includes/wikia/GlobalTitle.php

```diff
@@ -835,6 +844,32 @@ private function loadNamespaceNames() {
 	}
 
 	/**
+	 * Get $wgExtraNamespaces for title language
+	 *
+	 * @return array
+	 */
+	private function getLocalWgExtraNamespaces() {
```

Comment:
This is a little confusing as a method name with `LocalWgExtraNamespaces` given we also have the variable `wgExtraNamespacesLocal`, how about something like `getExtraExtensionNamespaces`?